### ISSUE TITLE
Call by reference in ab13bd

### ISSUE THREAD

**Issue by KybernetikJo** [python-control/Slycot#199]
State space matrices A,B,C,D are changed after call.

```python
    out = _wrapper.ab13bd(dico, jobn, n, m, p, A, B, C, D, tol)
```

Fix, shallow copy
```python
    a = A.copy()
    b = B.copy()
    c = C.copy()
    d = D.copy()

    out = _wrapper.ab13bd(dico, jobn, n, m, p, a, b, c, d, tol)
```

**Comment by KybernetikJo** [comment]
```python
def h2norm_slycot_c(A,B,C,D):
    dico = 'C'
    jobn = 'H'

    n, m = B.shape
    p, _ = C.shape

    #a = A.copy()
    #b = B.copy()
    #c = C.copy()
    #d = D.copy()

    a = A
    b = B
    c = C
    d = D

    h2norm = 0
    h2norm = slycot.ab13bd(dico, jobn, n, m, p, a, b, c, d)
    return h2norm

A1 = np.array([[0.0, 1.0],[-0.5, -0.1]])
B1 = np.array([[0.],[1.]])
C1 = np.eye(2)
D1 = np.zeros((2,1))

print(h2norm_slycot_c(A1,B1,C1,D1))
print(h2norm_slycot_c(A1,B1,C1,D1))
print(h2norm_slycot_c(A1,B1,C1,D1))

# 3.872983346207419
# 3.838232974288851
# 3.7856002470901817
```
Fix, shallow copy
```python
def h2norm_slycot_c(A,B,C,D):
    dico = 'C'
    jobn = 'H'

    n, m = B.shape
    p, _ = C.shape

    a = A.copy()
    b = B.copy()
    c = C.copy()
    d = D.copy()

    #a = A
    #b = B
    #c = C
    #d = D

    h2norm = 0
    h2norm = slycot.ab13bd(dico, jobn, n, m, p, a, b, c, d)
    return h2norm

A1 = np.array([[0.0, 1.0],[-0.5, -0.1]])
B1 = np.array([[0.],[1.]])
C1 = np.eye(2)
D1 = np.zeros((2,1))

print(h2norm_slycot_c(A1,B1,C1,D1))
print(h2norm_slycot_c(A1,B1,C1,D1))
print(h2norm_slycot_c(A1,B1,C1,D1))

# 3.872983346207419
# 3.872983346207419
# 3.872983346207419
```

**Comment by roryyorke** [comment]
For future readers: see #200 for further discussion.

**Comment by bnavigator** [comment]
Closed by #200, the general issue is still open.
